apps/create-plugin/template/src/test-transactions.ts.t

```diff
@@ -0,0 +1,65 @@
+{{#if shouldIncludeGreaterThanOrEqual}}
+import { 
+  type {{capitalize actionType}}ActionParams, 
+  GreaterThanOrEqual,
+} from '@rabbitholegg/questdk'
+{{else}}
+import { type {{capitalize actionType}}ActionParams } from '@rabbitholegg/questdk'
+{{/if}}
+
+{{#if tx.length}}
+import {
+  createTestCase,
+  type TestParams,{{#eq actionType 'bridge'}}
+  Chains{{/eq}}
+} from '@rabbitholegg/questdk-plugin-utils'
+{{else}}
+import { type TestParams } from '@rabbitholegg/questdk-plugin-utils'
+{{/if}}
+
+{{#unless tx.length}}
+export const {{uppercase actionType}}_TEST: TestParams<{{capitalize actionType}}ActionParams> = {
+  transaction: {
+    chainId: '',
```

Comment:
can we set these with valid typed default values so it compiles if the user doesn't provide a tx?

Reply:
FIxed!